shell/security/prismaci/twist-scan.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+#
+# This script scans docker images for vulnerabilities, generates local
+# report and also uploads the report to Prisma Cloud. The script expects
+# special credentials - those are set by prismacloud-credentials circleci context.
+#
+
+## Add some safeties for external dependencies when running
+if [ -z "${CIRCLECI}" ]; then
+  echo "I can only run in CircleCI"
+  exit 1
+fi
```

Comment:
style: Use double brackets where possible, also `${}` is not really needed in general, try to use only when needed
```suggestion
if [[ -z $CIRCLECI ]]; then
  echo "I can only run in CircleCI"
  exit 1
fi
```